main_test.go

```diff
@@ -16,6 +16,7 @@
 package main
 
 import (
+	"github.com/stretchr/testify/assert"
```

Comment:
this import should be moved below as a separate block

Reply:
I guess you mean code orders like this

```go
import (
	"context"
	"fmt"
	"strings"
	"testing"
	"time"

	"github.com/prometheus/client_golang/prometheus/testutil"
	"github.com/sirupsen/logrus"
	"github.com/stretchr/testify/assert"
	"go.mongodb.org/mongo-driver/bson"

	"github.com/percona/mongodb_exporter/internal/tu"
)
```
changed, years ago I did the same order like this, when the Goland and gofmt auto changes dependency order, I thought the auto reordered was more go-style.